runtime/src/lib.rs

```diff
@@ -720,7 +736,8 @@ construct_runtime!(
 		Rewards: crml_staking_rewards::{Module, Call, Storage, Config, Event<T>} = 29,
 		Bounties: pallet_bounties::{Module, Call, Storage, Event<T>} = 30,
 		Tips: pallet_tips::{Module, Call, Storage, Event<T>} = 31,
-		Contracts: pallet_contracts::{Module, Call, Config<T>, Storage, Event<T>},
+		Contracts: pallet_contracts::{Module, Call, Config<T>, Storage, Event<T>} = 32,
+		Nft: crml_nft::{Module, Call, Storage, Event<T>} = 33,
```

Comment:
Let's keep NFT at `30`, as it already is in runtime v40

Reply:
Done